lib/container_orchestrator/object_definition.rb

```diff
@@ -176,9 +178,24 @@ def default_environment
         {:name => "WORKER_HEARTBEAT_FILE",   :value => Rails.root.join("tmp/worker.hb").to_s},
         {:name => "WORKER_HEARTBEAT_METHOD", :value => "file"},
         {:name => "TERRAFORM_RUNNER_URL",    :value => "https://opentofu-runner:6000"},
+        {:name => "SECRET_KEY", :value => opentofu_runner_secret_key},
+        {:name => "TERRAFORM_RUNNER_TOKEN", :value => opentofu_runner_token},
```

Comment:
This should also probably be volume mount to avoid the same security scan issue.

Reply:
We could probably avoid an ENV var or volume entierly if the token can be generated from the v2_key since all workers have access to that.

Reply:
True, but then every worker needs to load `jwt`.  Unless the require is moved inside the connection logic.